index.html

```diff
@@ -20,27 +22,160 @@
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+
+
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
 
-    <section class="content">
+      <article class="card-3">
+        <div class="card-content"></div>
+        <img src="./images/tonic2.png" alt="Tonic" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Tonic</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+
+      <article class="card-4">
+        <div class="card-content"></div>
+        <img src="./images/multipost2.png" alt="Multi-Post Project" class="project-image">
+        <div class="card-headers">
+          <h2><a href="#" class="project-title">Multi-Post Project</a></h2>
+          <div class="card-categories">
+            <a href="#" class="main-category">Canopy</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="sec-category">Back End Dev</a>
+            <img src="./resources/Counter.svg" alt="counter" class="counter-card">
+            <a href="#" class="date-category">2015</a>
+          </div>
+        </div>
+        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+        <div class="hashtag-card">
+          <a href="#" class="hashtag">html</a>
+          <a href="#" class="hashtag">css</a>
+          <a href="#" class="hashtag">javaScript</a>
+        </div>
+        <button type="button" class="see-project">See Project</button>
+      </article>
+    </div>
+    <div class="about">
       <h1 class="title">
```

Comment:
- [ ] Replace `h1` with a `h2` tag. The `h1` tag is just for the title of the website.